src/property-filter/i18n-utils.ts

```diff
@@ -177,39 +157,45 @@ export function usePropertyFilterI18n(def: I18nStrings & I18nStringsExt): I18nSt
       const formatter = i18n(
         'i18nStrings.removeTokenButtonAriaLabel',
         def.removeTokenButtonAriaLabel,
-        format => () => format(toI18n(token, { ariaLabel: true }))
+        format => () => format({ token__formattedText: formatToken(toFormatted(token)) })
```

Comment:
Before the change, the `removeTokenButtonAriaLabel` label required token property label, operator, and value fields. Now, only formatted text is required.